core/modules/cart/helpers/createMethodsData.ts

```diff
@@ -0,0 +1,44 @@
+import ShippingDetails from '@vue-storefront/core/modules/checkout/types/ShippingDetails'
+import { ShippingMethodsData } from '@vue-storefront/core/modules/cart/types/ShippingMethodsData'
+
+const getShipping = (shippingMethods: any[]) => {
```

Comment:
it would be nice to have real shippingMethod type instead of any, also name is not that clear
```suggestion
const getDefaultShippingMethod = (shippingMethods: any[] = []) => {
```